CHANGELOG-4.0.md

```diff
@@ -6,7 +6,7 @@
 
 ## Fixed
 - Fixed `Phalcon\Mvc\Model\Query\Builder::getPhql` to add single quote between string value on a simple condition. [#14874](https://github.com/phalcon/cphalcon/issues/14874) [@jenovateurs](https://github.com/jenovateurs)
-- Fixed schema manipulation in `Phalcon\Db\Dialect\Mysql`, remove unnecessary `NULL` in column definition, unquote numerical defaults. [#14888](https://github.com/phalcon/cphalcon/pull/14888) [@pfz](https://github.com/pfz)
+- Fixed schema manipulation in `Phalcon\Db\Dialect\Mysql`, unquote numerical defaults. [#14888](https://github.com/phalcon/cphalcon/pull/14888) [@pfz](https://github.com/pfz)
```

Comment:
Add change log entry if proposed changes will affect users

Reply:
Moved this from fixed to changed. Nothing won't be affected to end user. Just more clean generated SQL code.